odoo/addons/test_lint/tests/test_pylint.py

```diff
@@ -67,9 +68,10 @@ def test_pylint(self):
             '--enable=%s' % ','.join(self.ENABLED_CODES),
             '--reports=n',
             "--msg-template='{msg} ({msg_id}) at {path}:{line}'",
-            '--load-plugins=pylint.extensions.bad_builtin,_odoo_checkers',
+            '--load-plugins=pylint.extensions.bad_builtin,_odoo_checkers,_odoo_checker_sql_injection',
             '--bad-functions=%s' % ','.join(self.BAD_FUNCTIONS),
-            '--deprecated-modules=%s' % ','.join(self.BAD_MODULES)
+            '--deprecated-modules=%s' % ','.join(self.BAD_MODULES),
+            '--ignore=tests',
```

Comment:
FYI I just ignored tests since that a sql-injection was detected here.
 - https://github.com/odoo/enterprise/blob/745649dfc/website_crm_score/tests/common.py#L93

IMHO these kind of pylint checks could be skipped for `tests` folder.

If you like run checks for tests folder so I can ignore it directly from sql-injection check.

Reply:
This query can actually be fixed in the test.